src/pages/GraduationCalculatorPage/components/GeneralCourse/GeneralCourseListModal/index.tsx

```diff
@@ -22,10 +22,11 @@ function GeneralCourseListModal({
   const generalCourseLectures = generalCourses?.lectures ?? [];
 
   const tableData = generalCourseLectures.map((lecture) => [
-    <span key={`name-${lecture.id}`}>{lecture.name}</span>,
-    <span key={`professor-${lecture.id}`}>{ }</span>,
-    <span key={`grades-${lecture.id}`}>{lecture.grades}</span>,
-    <span key={`courseType-${lecture.id}`}>교양선택</span>,
+    <span>{lecture.name}</span>,
+    <span>{ }</span>,
```

Comment:
```suggestion
    <span>{ }</span>, 
    {/* ex) 교수님 성함이 들어가던 자리 ~~이유로 비워져있음*/}
```
위 같은 부분은 주석을 달아주면 레거시로 남아도 코드를 파악하는데 더 수월할 것 같아요!

Reply:
넵! 교수명 및 삭제버튼을 비운 이유에 대해 간단하게 주석 달아놓았습니다! 🙇‍♀️

69e1d9607e562b5f294410aea24027d380f40f8d